bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/CPU.java

```diff
@@ -2508,6 +2528,62 @@ private static void handleVariableUnlock(WorkerExecutionContext ctx, BType[] typ
         }
     }
 
+    private static BRefType<?> execListGetOperation(BNewArray array, long index) {
+        switch (array.getTag()) {
+            case TypeTags.BOOLEAN_ARRAY_TAG:
```

Comment:
We can use the TypeTag of the elementType of the array, instead of creating a new TypeTags set